include/freerdp/server/rail.h

```diff
@@ -140,7 +140,7 @@ extern "C"
 
 	FREERDP_API RailServerContext* rail_server_context_new(HANDLE vcm);
 	FREERDP_API void rail_server_context_free(RailServerContext* context);
-	FREERDP_API UINT rail_server_handle_messages(RailServerContext* context);
```

Comment:
Not a good idea if you want a backport of this ;)

Reply:
Hmm okay, although I don't like that this is exported :/ it really shouldn't be, I'll create an issue for that and we'll address it in the next major?